src/tests/intg/test_sssctl.py

```diff
@@ -359,3 +361,28 @@ def test_group_show_basic_fqname_insensitive(ldap_conn,
     output = get_call_output(["sssctl", "group-show", "camelcasegroup1@LDAP"])
     assert output.find("Name: camelcasegroup1@LDAP") != -1
     assert output.find("Cached in InfoPipe: No") != -1
+
+
+@pytest.fixture
+def add_tripled_netgroup(request, ldap_conn):
+    ent_list = ldap_ent.List(ldap_conn.ds_inst.base_dn)
+
+    ent_list.add_netgroup("tripled_netgroup", ["(host,user,domain)"])
+
+    ent_list.add_netgroup("adv_tripled_netgroup", ["(host1,user1,domain1)",
+                                                   "(host2,user2,domain2)"])
+
+    create_ldap_fixture(request, ldap_conn, ent_list)
+    return None
+
+
+def test_netgroup_show(ldap_conn,
+                       sanity_rfc2307,
+                       portable_LC_ALL,
+                       add_tripled_netgroup):
+    res, _, netgrps = sssd_netgroup.get_sssd_netgroups("tripled_netgroup")
+    assert res == sssd_netgroup.NssReturnCode.SUCCESS
+    assert netgrps == [("host", "user", "domain")]
+
+    output = get_call_output(["sssctl", "netgroup-show", "tripled_netgroup"])
+    assert output.find("Name: tripled_netgroup") != -1
```

Comment:
I think it simpler to read following version:
```
assert "Name: tripled_netgroup" in output
```
And it looks more pythonic to me

And you might also check message for netgroup which is not in cache
"Netgroup adv_tripled_netgroup is not present in cache.". You might use different netgroup
or the same before `sssd_netgroup.get_sssd_netgroups` and then you can remove `adv_tripled_netgroup` from setup